torch/_functorch/aot_autograd.py

```diff
@@ -934,9 +959,12 @@ def inner_fn(*args):
         ]
 
         intermediate_bases = []
-        for o, info in zip(outs, meta.output_info):
+        for i, (o, info) in enumerate(zip(outs, meta.output_info)):
             if info.output_type == OutputType.alias_of_intermediate_save_as_output:
                 intermediate_bases.append(o._base)
+            elif info.output_type == OutputType.unsafe_view_alias:
+                # See Note [Intermediate Bases Optimization]
+                outs[i] = torch.ops.aten._unsafe_view(o, o.shape)
```

Comment:
I don't think this matters too much, because this is run at trace time (and we'll still end up baking the overload into the graph). But good to know - I'll update.

Reply:
Compilation time is still something we want to keep down no? :D

Reply:
agreed 😛 (to be far this op will be called usually a handful of times per compiled subgraph, so pretty infrequently (at most, # graph outputs, if every output is an alias of a graph intermediate))